pkg/cli/cmd/run/run.go

```diff
@@ -159,28 +159,56 @@ func (r *Runner) Run(ctx context.Context) error {
 		return clierrors.Message("Only kubernetes runtimes are supported.")
 	}
 
-	// We start three background jobs and wait for them to complete.
+	applicationSelector, err := portforward.CreateLabelSelectorForApplication(r.ApplicationName)
+	if err != nil {
+		return err
+	}
+
+	dashboardSelector, err := portforward.CreateLabelSelectorForDashboard()
+	if err != nil {
+		return err
+	}
+
+	// We start 5 background jobs and wait for them to complete.
 	group, ctx := errgroup.WithContext(ctx)
 
-	// 1. Display port-forward messages
-	status := make(chan portforward.StatusMessage)
+	// 1. Display port-forward messages for application
+	applicationStatusChan := make(chan portforward.StatusMessage)
+	group.Go(func() error {
+		r.displayPortforwardMessages(applicationStatusChan)
+		return nil
+	})
+
+	// 2. Display port-forward messages for dashboard
+	dashboardStatusChan := make(chan portforward.StatusMessage)
```

Comment:
what happen if user doesn't install dashboard?

Reply:
The portforward attempt will end and report to the user that the dashboard is not installed: https://github.com/radius-project/radius/pull/7186#discussion_r1500994822